test/EndToEnd/tests/LegacyCsprojPackageRefTests.ps1

```diff
@@ -0,0 +1,27 @@
+# basic create for uwp package ref based project
+function Test-UwpPackageRefClassLibraryCreate {
+
+    # Arrange & Act
+    $project = New-UwpPackageRefClassLibrary UwpLibrary1
+
+    # Assert
+    Assert-NetCoreProjectCreation $project
```

Comment:
will do the cleanup in a later PR. no point adding a new function just for changing the name.